README.md

```diff
@@ -1,46 +1,29 @@
-## 18F Guides Template
+## Joining 18F
 
-This is a skeleton repo containing the
-[CFPB/DOCter](https://github.com/CFPB/DOCter)-based
-[Jekyll](http://jekyllrb.com/) template for
-[18F Guides](http://18f.github.io/guides/).
+This is the repository for the page about [Joining 18F](https://pages.18f.gov/joining-18f/). This is an [18F Guide](https://github.com/18F/guides), built on the [18F Guides Template](https://github.com/18F/guides-template).
 
-### Generating the site/hosting locally
+The application form itself lives in a separate repository: https://github.com/18F/joining-18f-app.
```

Comment:
Ah, I misinterpreted your question the other day. The version of the application form at https://github.com/18F/joining-18f-app is on indefinite hold at the moment. The current application form is a Google form.